Makefile

```diff
@@ -573,6 +579,41 @@ run-android-benchmark-$1-%: android-benchmark-$1
 	rm -rf $(MBGL_ANDROID_BENCHMARK_DIR)/results && mkdir -p $(MBGL_ANDROID_BENCHMARK_DIR)/results
 	adb pull $(MBGL_ANDROID_LOCAL_BENCHMARK_DIR)/results.json $(MBGL_ANDROID_BENCHMARK_DIR)/results > /dev/null 2>&1
 
+# Run render tests for specified abi
```

Comment:
We are moving away from using the Makefile with the build system rework (see https://github.com/mapbox/mapbox-gl-native/tree/master/next). Might make more sense putting this code inside a script from the get-go as this code won't be migrated to the new downstream mapbox-gl-native-android repository.

cc @tmpsantos

Reply:
Yes, it is true. Currently I only made the render test running with old build system. And the work for making it running with new building system as a simple App is still ongoing. @tmpsantos also mentioned that it is not necessary to adapt the old build system.